Organization/Organization/CreateEmployeeViewController.h

```diff
@@ -0,0 +1,15 @@
+//
+//  CreateEmployeeViewController.h
+//  Organization
+//
+//  Created by New user on 02.08.17.
+//  Copyright © 2017 New user. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "Employee.h"
+
+@interface CreateEmployeeViewController : UIViewController
+@property (readwrite) Employee* savedEmployee;
```

Comment:
You shouldn't specify readwrite, but you should specify memory management field and atomicity.